### PR TITLE
Add Zcoin icon to Readme

### PR DESCRIPTION
**Pull request by sctnightcore**
## PR intention
DeleteMe: Mandatory: What this PR is intended to do, what change it introduces, what issue it solves.

## Code changes brief
DeleteMe: Optional: What are the architectural, UX, or other kind of changes are made which are hard to induce from the code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Zcoin
+<img src="https://zcoin.io/storage/2017/12/ci_logo_monochrome.svg" width="500">
```
Comment (ultimaweapon) [inline README.md]: @reubenyap do we have a better URL than this one?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 ===============
```
Comment (ultimaweapon) [inline README.md]: This line should be removed too.

Comment (sctnightcore) [reply README.md]: =============== @ultimaweapon 
this ?

File: README.md (hunk 1 above)
Comment (sctnightcore) [reply README.md]: I think size is ok for me .

File: README.md (hunk 2 above)
Comment (sctnightcore) [reply README.md]: Fixed : 7d1be49

File: README.md (hunk 1 above)
Comment (reubenyap) [reply README.md]: What is the issue with the URL?

Comment (ultimaweapon) [reply README.md]: @reubenyap the problems with this URL is it look like it is managed by CMS. So it is better to use the one that we uploaded to the web server by ourself.

Comment (riordant) [reply README.md]: @ultimaweapon where can we find that URL?

Comment (ultimaweapon) [reply README.md]: Do you mean the one that we uploaded by ourself? If yes, no I don't have.

Comment (ultimaweapon) [reply README.md]: @reubenyap can we upload this image to our website with a proper location?

**Comment by reubenyap** [comment]
No longer needed since rebrand to Firo

**Pull request closed without merging**